Packs/CommonScripts/Scripts/GetEntries/GetEntries_test.py

```diff
@@ -0,0 +1,74 @@
+from CommonServerPython import *

```

Comment:
Let's add "given-when-then" documentation to this tests cases.

Reply:
@YuvHayun Could you paste the screenshot of what the error raised?